components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Models/EventMonitoredItemModel.cs

```diff
@@ -0,0 +1,59 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.Azure.IIoT.OpcUa.Protocol.Models {
+    using Microsoft.Azure.IIoT.OpcUa.Core.Models;
+
+    /// <summary>
+    /// Data monitored item
+    /// </summary>
+    public class EventMonitoredItemModel : BaseMonitoredItemModel {
+        /// <summary>
+        /// Event filter
+        /// </summary>
+        public EventFilterModel EventFilter { get; set; }
+
+        /// <summary>
+        /// Clone
+        /// </summary>
+        /// <returns></returns>
+        public override BaseMonitoredItemModel Clone() {
```

Comment:
Would be good to write tests now that we have changed the Clone() a bit. Can we add (or track)?

Reply:
I will write unit tests for MonitoredItemModel in this PR